src/components/Api.js

```diff
@@ -0,0 +1,31 @@
+class Api {
+  constructor(options) {
+    this.baseUrl = options.baseUrl,
+    this.headers = options.headers
+  }
+
+  postPledge({Name, Surname, Email, Pledge}) {
+    return fetch(`${this.baseUrl}/pledges`, {
+      method: "POST",
+      headers: this.headers,
+
+      body: JSON.stringify({
+        Name,
```

Comment:
you should make these lowercase since it's not a class

Reply:
This is what Alex posted as an example: {
	"Name": "Alex",
	"Surname": "Popkov",
	"Email": "poalrom@yandex.ru",
	"Pledge": "15"
}